go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/ppc64le-cloud/packer-plugin-powervs
 
-go 1.18
+go 1.19
 
 require (
 	github.com/IBM-Cloud/power-go-client v1.6.0
 	github.com/IBM/go-sdk-core/v5 v5.17.0
 	github.com/IBM/platform-services-go-sdk v0.62.7
 	github.com/hashicorp/hcl/v2 v2.20.1
-	github.com/hashicorp/packer-plugin-sdk v0.4.0
```

Comment:
please don't bump the version to 0.5.2 because there is licence change in the recent version!

Reply:
Which version should I use then, I see with v0.4.0, I was hitting nil pointer error during make generate

Reply:
Any latest v0.4.0 version but not 0.5.x

Reply:
wait! I don't see licence file change for this package which is good, lets merge with this version update.